src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/LdapDirectoryDAO.scala

```diff
@@ -52,7 +52,7 @@ class LdapDirectoryDAO(
   override def loadGroup(groupName: WorkbenchGroupName): IO[Option[BasicWorkbenchGroup]] = {
     val res = for {
       entry <- OptionT(executeLdap(IO(ldapConnectionPool.getEntry(groupDn(groupName)))).map(Option.apply))
-      r <- OptionT.liftF(IO(unmarshalGroupThrow(entry)))
+      r <- OptionT.liftF(unmarshalGroupThrow(entry))
```

Comment:
I will avoid using these `throw` methods with fs2...why not `IO.fromEither(unmarshalGroup(entry))`